index.js

```diff
@@ -322,6 +322,17 @@ module.exports = {
                   format: [ 'snake_case' ],
                   leadingUnderscore: 'require',
                },
+               {
+                  selector: 'classProperty',
+                  modifiers: [ 'private', 'readonly', 'static' ],
+                  format: [ 'UPPER_CASE' ],
+                  leadingUnderscore: 'require',
+               },
+               {
+                  selector: 'classProperty',
+                  modifiers: [ 'public', 'readonly', 'static' ],
```

Comment:
Do we need `protected` `readonly` `static`?

Reply:
You're right!

Reply:
Done